src/panels/config/blueprint/ha-blueprint-overview.ts

```diff
@@ -114,8 +114,14 @@ class HaBlueprintOverview extends LitElement {
             ? ""
             : html`<mwc-icon-button
                 .blueprint=${blueprint}
+                .label=${this.hass.localize(
+                  "ui.panel.config.blueprint.overview.use_blueprint"
+                )}
+                title=${this.hass.localize(
+                  "ui.panel.config.blueprint.overview.use_blueprint"
+                )}
                 @click=${(ev) => this._createNew(ev)}
-                ><ha-svg-icon .path=${mdiPlus}></ha-svg-icon
+                ><ha-svg-icon .path=${mdiRobot}></ha-svg-icon
```

Comment:
Let's make this a text button instead of an icon, I don't think this is clear.

Reply:
Changed and adjusted screenshot on top.